src/Network/Network/Network.format.ps1xml

```diff
@@ -2526,11 +2526,11 @@
                 <PropertyName>IpAddress</PropertyName>
               </ListItem>
               <ListItem>
-                <Label>NetworkInterfaceIpConfiguration</Label>
+                <Label>NetworkInterfaceIpConfigurationId</Label>
                 <PropertyName>NetworkInterfaceIpConfigurationText</PropertyName>
               </ListItem>
               <ListItem>
-                <Label>VirtualNetwork</Label>
+                <Label>VirtualNetworkId</Label>
                 <PropertyName>VirtualNetworkChildResourceText</PropertyName>
```

Comment:
shouldn't this be VirtualNetworkIdText

Reply:
No, that label is what is printed on the powershell console. I think it would read kind of weird to add 'text' to the output console

Reply:
Ohh are you referring to VirtualNetworkChildResourceText? Yeah that property name isn't a big deal, I suppose it should be but its fine as is, its not visible to any user.